CONTRIBUTING.md

```diff
@@ -81,15 +81,15 @@ The DCO is a declaration attached to every contribution made by every developer
 
 **Each commit must include a DCO which looks like this:**
 
-```txt
+"`txt
```

Comment:
We are creating a codeblock here. The requested change here is incorrect.

Reply:
Fixed